Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN npm config set cafile "/app/DOIRootCA2.cer" \
 
 RUN node_modules/bower/bin/bower install --allow-root
 
+RUN nosetests --all-modules --exe
```

Comment:
I wonder if this is a Windows issue. I know Andrew Yan has run nosetest on his Windows machine without having to use that flag. It might be worthwhile pinging him.